test/extended/clusterup.sh

```diff
@@ -292,37 +260,47 @@ readonly default_tests=(
     "portinuse"
     "svcaccess"
     "persistentvolumes"
-
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "default"
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "image::ose3.3"
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "image::ose3.4"
-# enable once https://github.com/openshift/origin/issues/16995 is fixed
-#    "image::ose3.5"
-
-    "image::ose3.6"
+    "default"
 
 # logging+metrics team needs to fix/enable these tests.
 #    "metrics"
 #    "logging"
 )
 
+# run each test with each of these set of additiona args.  Primarily
+# intended to run the tests against different cluster versions.
+readonly extra_args=(
+    # Test the code being delivered
+    "--loglevel=2 --version=${ORIGIN_COMMIT}"
+    # Test the current published release
+    "--loglevel=2"  # can't be empty, so pass something benign
+    # Test the previous origin release
+    "--loglevel=2 --image=docker.io/openshift/origin --version=v3.7.0-rc.0"
+    # Test the previous OCP release
+    # TODO - turn this on once OCP v3.7 is available.
+    #   "--image=registry.access.redhat.com/openshift3/ose --version=v3.7"
+)
 tests=("${1:-"${default_tests[@]}"}")
 
 ORIGIN_COMMIT=${ORIGIN_COMMIT:-latest}
 
+# re-tag the latest service catalog image w/ the origin commit because we didn't
```

Comment:
@stevekuznetsov I expected to build this, not hard-code a pull of `latest`

Reply:
same comment as the other thread...  I agree with you, but imho that means the job that all our extended test jobs inherit from should be building all the "external" images we now have (registry, service catalog, presumably others) and tagging them, doing it in each extended test job is going to be error prone at best.

Reply:
Yeah, @csrwng and I are trying to make sure this look sane for the jobs we write as pipelines but today until we are there we need some sort of band-aid (tourniquet?) and this seems ... ok?

Reply:
Wait, hold on -- misread this. I would expect the versioning to be top-down driven at all places in our tests. We need to have a clear way to manage that from the job orchestration level.

Reply:
yes, agreed, but until we do this is the simplest thing to allow our tests to proceed in at least some useful fashion.